app/styles/navbar.module.css

```diff
@@ -231,15 +231,15 @@ nav li {
     float: none;
   }
 
-  .nav__home {
-    display: none;
-  }
-
   .nav__home__link,
   .active {
     display: block;
   }
 
+  .nav__home {
+    display: none;
+  }
```

Comment:
Why display none

Reply:
It was added before if you will notice we don't have logo in mobile navbar that's why `display: none`

Reply:
Here display none is applied to 1024 screen size

Reply:
yes